quarkus.jdt/com.redhat.quarkus.jdt.test/src/main/java/com/redhat/quarkus/jdt/core/JDTQuarkusManagerTest.java

```diff
@@ -31,6 +34,10 @@
 
 	private static final String QUARKUS_CORE_DEPLOYMENT_JAR = getJarPath("quarkus-core-deployment-0.19.1.jar.test");
 
+	static {

```

Comment:
same thing. If you need it for multiple classes, then better to use a base test class that fiddles with log levels

Reply:
Done with new class BaseJDTQuarkusManagerTest